### PR TITLE
Optional Digital input counter & reset resource created in Presence IPSO object

### PR DESCRIPTION
**Pull request by thetoster**
Signed-off-by: bartlomiej.zarnowski <bartlomiej.zarnowski@imgtec.com>

### CODE REVIEW

File: lwm2m-client-ipso-presence.c [hunk 1]
```diff
@@ -182,6 +182,18 @@ AwaError PresenceObject_DefineObjectsAndResources(AwaStaticClient *awaClient, Aw
     }
 
     AwaStaticClient_CreateObjectInstance(awaClient, IPSO_PRESENCE_OBJECT, 0);
+    error = AwaStaticClient_CreateResource(awaClient, IPSO_PRESENCE_OBJECT, 0, IPSO_PRESENCE_DIGITAL_INPUT_COUNTER_RESET);
+    if (error != AwaError_Success)
```
Comment (abhijit-mahajani-imgtec) [inline lwm2m-client-ipso-presence.c]: It was not there earlier. i am getting 
 [ERROR] [lwm2m_client_core.c:1218] Read on 3302/0/5505 failed, handler returned 500 (expected 205)
at the startup of motion-sensor app.

however if i remove creation of IPSO_PRESENCE_DIGITAL_INPUT_COUNTER_RESET then do not get this warning.

**Comment by abhijit-mahajani-imgtec** [comment]
commit message can be changed to mention creation of resource

**Comment by abhijit-mahajani-imgtec** [comment]
But in general motion-sensor app is working fine with motion-led-controller.

**Comment by nikhil-zinjurde-imgtec** [comment]
test with jenkins - dev

File: lwm2m-client-ipso-presence.c (hunk 1 above)
Comment (thetoster) [inline lwm2m-client-ipso-presence.c]: Fixed & ammended

**Comment by nikhil-zinjurde-imgtec** [comment]
test with jenkins - dev

**Comment by abhijit-mahajani-imgtec** [comment]
Approved and verified

**Pull request merged**